Ensure that we fail if libpcap can't be detected, rather than exiting with a success return code.
--- a/configure.seed
+++ b/configure.seed
@@ -126,12 +126,9 @@ case "$host" in
          AC_CHECK_LIB([dl], [dlopen, dlsym],   [DL_LIB="-ldl"],[AC_CHECK_LIB([c], [dlopen, dlsym], [DL_LIB="-lc"],[AC_MSG_ERROR([unable to find the dlopen(), dlsym() functions]) ]) ])
       else
          AC_CHECK_LIB([pcap], [pcap_open_live], [PCAP_LIB="-lpcap"])
-	 if test $ac_cv_lib_pcap_pcap_open_live = "no"; then :
-            echo ""
-            echo "ERROR: Missing libpcap(-dev) library required to compile the example application"
-            echo "ERROR: Please install it and try again"
-            exit
-	 fi
+         if test $ac_cv_lib_pcap_pcap_open_live = "no"; then :
+            AC_MSG_ERROR([Cannot find libpcap which is required to compile the example application])
+         fi
       fi
       ;;
 esac
